requirements.txt

```diff
@@ -4,7 +4,7 @@
 numpy
 flake8==3.8.3
 pylint==2.5.3
-pytest==6.0.0
+pytest>=6.0.0
```

Comment:
Why do you need the inequality here? Usually we prefer to pin as much as possible to avoid unexpected changes breaking compatibility (although there are lots of unpinned things in this set of requirements).

Reply:
agree good point, It was set to 6.0 and tested last one quicly, then forgot to pin.